content/docs/studio/rest-api.md

```diff
@@ -103,6 +103,144 @@ Running this example will fail because it requires that you have [set up remote
 cloud credentials] in DVC Studio to where the model is stored.
 
 </admon>
+</toggle>
+
+## Create new project
+
+To create a new project programmatically, you can use with the DVC Studio API.
+The API provides a POST request endpoint for creating projects. Authentication
+is required using a token, which can be generated from the user's profile
+settings in DVC Studio
+
+```yaml
+Endpoint: api/create-project
+HTTP Method: POST
+Content-Type: application/json
+```
+
+### Request
+
+Here are the possible values for each parameter in the request json:
+
+- `repo_url` (required) - the URL of the repository where the project will be
+  created. This field is mandatory.
+- `public` (boolean, default: `false`) - specifies whether the project should be
+  public (`true`) or not (`false`).
+- `team_name`(string) - the name of the team associated with the project if we
+  want to create the project in team space. This field is optional. If team name
+  is not specified, the project will be in user namespace.
+- `subdir`(string) - the subdirectory within the repository where the project
+  will be located, in case of a monorepo. This field is optional.
+- `name` (string) - the name of the project to be defined. If no name is
+  specified, it will be extracted from repository name. This field is optional.
+
+The request should contain following header for authorization.
+
+| header        | desc            | example value |
+| ------------- | --------------- | ------------- |
+| Authorization | Header for auth | token abc123  |
```

Comment:
minor, put a realistic token

Reply:
let's also move it to a separate subsection "### Headers" or "### Authentication"